drivers/hdlc_rcp_if/hdlc_rcp_if_nxp.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2024, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * IEEE 802.15.4 HDLC RCP interface. This is meant for network connectivity
+ * between a host and a RCP radio device.
+ */
+
+/* -------------------------------------------------------------------------- */
+/*                                  Includes                                  */
+/* -------------------------------------------------------------------------- */
+#include <fwk_platform_hdlc.h>
+#include <fwk_platform_ot.h>
+#include <openthread/platform/radio.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <zephyr/init.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/net/hdlc_rcp_if/hdlc_rcp_if.h>
+#include <zephyr/net/ieee802154_radio.h>
+#include <zephyr/net/openthread.h>
+
+/* -------------------------------------------------------------------------- */
+/*                                  Definitions                               */
+/* -------------------------------------------------------------------------- */
+
+#define DT_DRV_COMPAT nxp_hdlc_rcp_if
+
+#define LOG_MODULE_NAME hdlc_rcp_if_nxp
+#define LOG_LEVEL       CONFIG_NET_OPENTHREAD_HOST_LOG_LEVEL
```

Comment:
I guess you should use `CONFIG_HDLC_RCP_IF_DRIVER_LOG_LEVEL` instead?

Reply:
Resolved